sycl/doc/extensions/ComplexAlgorithms/ComplexAlgorithms.asciidoc

```diff
@@ -0,0 +1,133 @@
+= SYCL_EXT_ONEAPI_COMPLEX_ALGORITHMS
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+== Introduction
+IMPORTANT: This specification is a draft.
+
+NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
+
+NOTE: This document is better viewed when rendered as html with asciidoctor.  GitHub does not render image icons.
```

Comment:
Please remove this note.  We think GitHub does render Asciidoctor files well.

Reply:
Fixed in ba78fd4764399f36179665ac62323ff358e4d2eb.